src/pages/index.js

```diff
@@ -1,11 +1,19 @@
 import Link from "next/link";
+import Image from "next/image";
 
 export default function Home() {
   return (
     <div>
-      <main>
+      <main className="home">
         <h1>Accessibility Cheatsheets for Components
         </h1>
+        <p className="home__header-description">
+          Intro copy goes here, such as: Build accessible components 
+          more accurately and in less time with these easy-to-read 
+          cheatsheets based on WCAG and ARIA best practices. 
+          Check back often as we’re adding new components frequently! 
+        </p>
+        <Image id="home__header-image" src="/home-header-image.svg" width="100%" height="100%" />
```

Comment:
🏆 Excellent setup here and great job grouping together like work into different commits. Well done!